frappe/tests/ui/test_calendarView.js

```diff
@@ -0,0 +1,56 @@
+QUnit.module('views');
+
+QUnit.test("Create and delete an event", function(assert) {
+	assert.expect(4);
+	let done = assert.async();
+	let randomText = frappe.utils.get_random(10);
+	let today = frappe.datetime.get_datetime_as_string();
+
+	frappe.run_serially([
+		
+		() => {
+			return frappe.tests.make("Event", [
+					{subject: randomText},
+					{starts_on: today},
+					{event_type: 'Private'}
+			]);
+		},
+
+		() => frappe.set_route(["List", "Event", "Calendar"]),
+
+
+		// Check if event is created
+		() => {
+			assert.deepEqual($('.fc-title').text(), randomText);
```

Comment:
Use `deepEqual` only for objects and arrays. Use `equal` or simple `===` for direct comparison